src/hackerspace_online/forms.py

```diff
@@ -37,11 +47,19 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['username'].help_text = 'Ask your teacher what you should be using for your username. Username is not case sensitive'
```

Comment:
While you are in here, can you update the help text?
`self.fields['username'].help_text = 'Username is not case sensitive.'